dotcom-rendering/src/web/components/WithABProvider.tsx

```diff
@@ -1,31 +1,59 @@
-import { CoreAPIConfig, OphanAPIConfig } from '@guardian/ab-core/dist/types';
-import { ABProvider } from '@guardian/ab-react';
+import { useEffect } from 'react';
+import { CoreAPIConfig } from '@guardian/ab-core/dist/types';
+import { ABProvider, useAB } from '@guardian/ab-react';
+import { getCookie, log } from '@guardian/libs';
+import { tests } from '@frontend/web/experiments/ab-tests';
```

Comment:
```suggestion
import { tests } from '../experiments/ab-tests';
```
in prep again for #3936